hakyll.cabal

```diff
@@ -252,7 +252,8 @@ Library
     Other-Modules:
       Hakyll.Web.Pandoc.Binary
     Build-Depends:
-      pandoc >= 2.11 && < 2.20 || >= 3.0 && < 3.7
+      pandoc >= 2.11 && < 2.20 || >= 3.0 && < 3.7,
```

Comment:
[pandoc 2.11](https://hackage.haskell.org/package/pandoc-2.11) through to 2.19 had support for pandoc-types 1.22; is there a reason to be more restrictive on the pandoc-types lower bound?

Reply:
Not really, I just put in whatever `cabal gen-bounds` gave me (which I realise is not a very scientific way to go about things)

Reply:
Understood, please allow pandoc-types-1.22 in this case, such that we don't necessarily (implicitly) restrict the version of pandoc to be < 3

Reply:
Thank you!